src/components/NewCommentForm.tsx

```diff
@@ -2,5 +1,80 @@
+import classNames from 'classnames';
+import React, { useState } from 'react';
+import { Comment } from '../types/Comment';
+
+type Props = {
+  addComment: (a: Comment) => Promise<void>;
```

Comment:
The `addComment` function should not expect a `Promise<void>` return type since it does not return a promise. Adjust the type definition accordingly.